packages/core/src/db/models/Report.ts

```
@@ -1,27 +1,140 @@
     public static async getReport(_id: string) {
       const report = await models.Reports.findOne({ _id });
       if (!report) {
-        throw new Error("Report not found");
+        throw new Error('Report not found');
       }
 
       return report;
     }
 
+    public static async getReports(params: IListParams, user: IUserDocument) {
+      const filter = await this.generateFilter(params, user);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)